exercises/practice/reverse-string/.meta/template-data.yaml

```diff
@@ -0,0 +1,27 @@
+properties:
+  reverse:
+    test: |-
+      sprintf(q:to/END/, %case<description>.raku, %case<input><value>.raku, %case<expected>.raku, %case<input><value>.raku, %case<input><value>.raku);
```

Comment:
I don't think you need to specify the value multiple times. The docs say indexing is NYI but it does seem to work when I tried it? https://docs.raku.org/routine/sprintf#NYI_Format_parameter_index_using_the_'$'_symbol